Dockerfile

```diff
@@ -51,17 +66,48 @@ RUN yum -y install \
     yum clean all && \
     rm -rf /var/cache/yum
 
+# Add hadoop repo. and install fetchdt
+ADD ./repos/hdp7-stable.repo /etc/yum.repos.d/hdp7-stable.repo
+RUN yum -y install \
+      hadoop-fetchdt && \
+    yum clean all && \
+    rm -rf /var/cache/yum
+
+# Add openstack repo. and install kubernetes-client, helm
+ADD ./repos/openstackclients7-queens-stable.repo /etc/yum.repos.d/openstackclients7-queens-stable.repo
+RUN yum -y install \
+      kubernetes-client \
+      helm && \
+    yum clean all && \
+    rm -rf /var/cache/yum
+
 # Upgrade pip package manager
 RUN pip3.6 install --upgrade pip
 
 # ----- Install JupyterHub ----- #
+
+# Install JupyterHub dependencies for postres db support, pycurl over https and cryptography for auth state
+RUN yum install -y \
+      libcurl-devel && \
+    yum clean all && \
+    rm -rf /var/cache/yum
+
+RUN PYCURL_SSL_LIBRARY=nss pip install \
```

Comment:
maybe do a break to make it more visible that first you are setting a env variable and then doing pip